contrib/zmq/zmq_sub.py

```diff
@@ -34,7 +34,7 @@
             print('- HASH BLOCK ('+sequence+') -')
             print(binascii.hexlify(body).decode("utf-8"))
         elif topic == "hashtx":
-            print ('- HASH TX ('+sequence+') -')
```

Comment:
Please remove the space here

Reply:
@nmarley there was a space originally, this commit removes that space. can revert this change if needed.

Reply:
@PaulieD You're right, sorry for getting the direction of change confused. This one is actually small, but personally I would still prefer to not mix formatting and functional PRs.

One reason is, what if this PR doesn't end up being merged in? That space fix would have to go in somewhere else then, and results in double-work (albeit minor).